src/main/java/ru/skypro/homework/controller/AdsController.java

```diff
@@ -0,0 +1,54 @@
+package ru.skypro.homework.controller;
+
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+import ru.skypro.homework.dto.Ad;
+import ru.skypro.homework.dto.Ads;
+import ru.skypro.homework.dto.CreateOrUpdateAd;
+import ru.skypro.homework.dto.ExtendedAd;
+
+@RestController
+@RequestMapping("/ads")
+@CrossOrigin(value = "http://localhost:3000")
+public class AdsController {
+
+    @GetMapping
+    public ResponseEntity getAds (){
+        return ResponseEntity.ok(new Ads());
+    }
+
+    @PostMapping(consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
+    public ResponseEntity postAds (@RequestBody CreateOrUpdateAd properties,
```

Comment:
```suggestion
    public ResponseEntity postAds (@RequestParam CreateOrUpdateAd properties,
```
или
```suggestion
    public ResponseEntity postAds (@RequestPart CreateOrUpdateAd properties,
```
иначе не будет работать)